[2025-09-08T17:12:51.415+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:12:51.422+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskD backfill__2025-05-17T12:00:00+00:00 [queued]>
[2025-09-08T17:12:51.427+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskD backfill__2025-05-17T12:00:00+00:00 [queued]>
[2025-09-08T17:12:51.427+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:12:51.439+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): taskD> on 2025-05-17 12:00:00+00:00
[2025-09-08T17:12:51.442+0000] {standard_task_runner.py:72} INFO - Started process 23141 to run task
[2025-09-08T17:12:51.445+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskD', 'backfill__2025-05-17T12:00:00+00:00', '--job-id', '246', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpxg2ukmrf']
[2025-09-08T17:12:51.446+0000] {standard_task_runner.py:105} INFO - Job 246: Subtask taskD
[2025-09-08T17:12:51.476+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskD backfill__2025-05-17T12:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:12:51.528+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskD' AIRFLOW_CTX_EXECUTION_DATE='2025-05-17T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2025-05-17T12:00:00+00:00'
[2025-09-08T17:12:51.528+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:12:51.539+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-08T17:12:51.540+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK D has executed!']
[2025-09-08T17:12:51.547+0000] {subprocess.py:86} INFO - Output:
[2025-09-08T17:12:51.549+0000] {subprocess.py:93} INFO - TASK D has executed!
[2025-09-08T17:12:51.549+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-08T17:12:51.564+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:12:51.565+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskD, run_id=backfill__2025-05-17T12:00:00+00:00, execution_date=20250517T120000, start_date=20250908T171251, end_date=20250908T171251
[2025-09-08T17:12:51.579+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:12:51.589+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:12:51.589+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
